src/amo/components/AddonMeta/index.js

```diff
@@ -39,14 +40,38 @@ export class AddonMetaBase extends React.Component<Props> {
     let reviewTitle;
     if (!addon) {
       reviewCount = null;
-      reviewTitle = i18n.gettext('Ratings');
-    } else if (addonRatingCount) {
-      reviewCount = i18n.formatNumber(addonRatingCount);
-      reviewTitle = i18n.ngettext('Rating', 'Ratings', addonRatingCount);
+      reviewTitle = i18n.gettext('Reviews');
+    } else if (addonReviewCount) {
+      reviewCount = i18n.formatNumber(addonReviewCount);
+      reviewTitle = i18n.ngettext('Review', 'Reviews', addonReviewCount);
     } else {
-      reviewTitle = i18n.gettext('No Ratings');
+      reviewTitle = i18n.gettext('No Reviews');
     }
 
+    const reviewsContent =
+      addon && reviewCount ? (
```

Comment:
could this be a variable since you use down below too? maybe something like `const hasReviews`...

Reply:
I tried, and Flow won't let me do that. If I don't explicitly specify `addon && reviewCount`, but rather put it into a `const`, Flow complains that `addon` can be null. I'm surprised about this, and Kumar agrees it seems like a Flow bug, but I don't know if there's much we can do about it.

Here's a [simple example](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVAXAngBwKZgCCAJsXAHZgC8YA3gD6phjECGGrAXHeawLZ5uAZwwAnAJbkA5gF8w9MOQCuMGABpU9GQG50AYwoiwGPEZoAKVqQrcSZSguWqAlNQB8dJmAPkjAC1YhABF2VmowK3swADJoiOtyADo2Dl1mcSgwcwDg0NdaLwBIUTwMJVFKSIpk0MTeATSwGVQdVCA) of the bug.

Reply:
ahh yep, I've had similar issues..